be/src/http/action/check_rpc_channel_action.cpp

```diff
@@ -38,7 +38,8 @@
 namespace doris {
 CheckRPCChannelAction::CheckRPCChannelAction(ExecEnv* exec_env, TPrivilegeHier::type hier,
                                              TPrivilegeType::type type)
-        : HttpHandlerWithAuth(exec_env, hier, type) {}
+        : HttpHandlerWithAuth(exec_env, hier, type), _exec_env(exec_env) {}
```

Comment:
There is already a Exec_env member in HttpHandlerWithAuth. Why not  change it to protected and delete the _exec_env member in CheckRPCChannelAction?